src/data/v2/stream.rs

```diff
@@ -326,13 +360,15 @@ impl subscribe::Message for ParsedMessage {
     match self {
       MessageResult::Ok(Ok(message)) => match message {
         DataMessage::Bar(bar) => subscribe::Classification::UserMessage(Ok(Ok(Data::Bar(bar)))),
+        DataMessage::Quotes(quote) => subscribe::Classification::UserMessage(Ok(Ok(Data::Quote(quote)))),
         DataMessage::Subscription(data) => {
           subscribe::Classification::ControlMessage(ControlMessage::Subscription(data))
         },
         DataMessage::Success => subscribe::Classification::ControlMessage(ControlMessage::Success),
         DataMessage::Error(error) => {
           subscribe::Classification::ControlMessage(ControlMessage::Error(error))
         },
+
```

Comment:
Please don't include spurious changes like this in a pull request.

Reply:
sorry